samples/attestation/host/host.cpp

```diff
@@ -150,6 +191,56 @@ int main(int argc, const char* argv[])
         return 1;
     }
 
+    if (argc == 5)
+    {
+        /* Set logging callback */
+        FILE* out_file = fopen(argv[4], "w");
+        oe_log_set_callback((void*)out_file, host_logging_callback);
+
+        int oe_log_level_size, azdcap_log_level_size;
+
+        /* Set logging levels */
+#if defined(__linux__)
+        /* Get current values of OE_LOG_LEVEL and AZDCAP_DEBUG_LOG_LEVEL */
+        char* oe_log_level = getenv(OE_LOG_LEVEL);
```

Comment:
Wouldn't the compiler complain about the re-definition?

Reply:
removed env variable setting logic inside code. Moved to CMakeList.txt.